pytorch_lightning/root_module/model_saving.py

```diff
@@ -45,7 +46,8 @@ def on_hpc_load(self, checkpoint):
 class TrainerIO(object):
 
     def __get_model(self):
-        is_dp_module = type(self.model) is LightningDistributedDataParallel or type(self.model) is LightningDataParallel
+        is_dp_module = isinstance(self.model, (LightningDistributedDataParallel,
```

Comment:
isinstance won't allow users to have custom subclasses. We need to keep it as type which allows users that flexibility

Reply:
what is the difference between yours "model is LightningDistributedDataParallel or model is LightningDataParallel" and now "model is one of LightningDistributedDataParallel or LightningDataParallel"?

Reply:
no, the difference is type vs isinstance.  

isinstance means it has to be that class and only that class.
type means it can be a child of that class, or child of child, etc...

Reply:
never mind, isinstance is the one we want.

Reply:
just realized the change was merging the or (which makes sense)